src/pages/Home.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Crypto, Herocrypto } from '../components';
+import { getData, selectFilteredCrypto } from '../redux/crypto/cryptoSlice';
+import '../index.css';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getData());
+  }, []);
+  const cryptoList = useSelector(selectFilteredCrypto);
+
+  return (
+    <div style={{ padding: '10px 10px 0 10px' }}>
+      <Herocrypto
+        name="Bitcoin"
+        id="bitcoin"
+        url="bitcoin"
+        symbol="btc"
+        value="20827.4673277614199303"
+      />
+      <span style={{ display: 'flex', color: '#dbdffd' }}>Stats by Rank</span>
+      <div className="cryptoList">
+        {cryptoList.slice(1).map((crypto) => (
+          <Crypto
+            name={crypto.name}
+            id={crypto.id}
+            url={crypto.id}
+            symbol={crypto.symbol.toLowerCase()}
+            value={crypto.priceUsd}
+            key={crypto.id}
+          />
+        ))}
+      </div>
+    </div>
```

Comment:
- [ ] You are yet to implement the **`alternating color pattern`** as we have on the design and as requested. All your tokens/coins on one side of the screen have a thicker background color, whereas in the design we have alternating backgrounds. Kindly re-arrange your patterns.

  In the design, for example, the second and third countries have thicker backgrounds; next, the fourth and fifth countries have lighter backgrounds, and so on.

  ![assad-2](https://user-images.githubusercontent.com/54848363/175789791-6718fbe2-8d2b-4da1-9a1f-37cd9a34dd2d.jpg)

Reply:
i cant get seem to be able to get the alternative pattern, could you offer a solution? @DammyShittu

Reply:
Hi @nottherealalanturing,
You can check this [link](https://www.w3schools.com/cssref/sel_nth-child.asp) to use the CSS Nth-child selector to get the alternative pattern as mentioned.

_Tip_

You can try this 👇 
```
<selector>:nth-child(4n + 2),
<selector>:nth-child(4n + 3)
```

I hope it helps. 😉